scripts/bin/xpedite

```diff
@@ -101,7 +101,10 @@ class AppController(ArgparseController):
   )
   def probes(self):
     from xpedite.profiler import Profiler
-    probes = Profiler.probes(self.app.pargs.profileInfo)
+    profileInfoPath = self.app.pargs.profileInfo
+    from xpedite.profiler.profileInfo import loadProfileInfo
+    profileInfo = loadProfileInfo(profileInfoPath)
+    probes = Profiler.probes(profileInfoPath)
```

Comment:
Thanks @sahmad98 for the bug report and patch. Good catch.

By convention the methods in the [profiler](https://github.com/Morgan-Stanley/Xpedite/blob/master/scripts/lib/xpedite/profiler/__init__.py) module accepts a file system path and loads the profileInfo object from it. This is followed in both [record](https://github.com/Morgan-Stanley/Xpedite/blob/master/scripts/lib/xpedite/profiler/__init__.py#L159) and [report](https://github.com/Morgan-Stanley/Xpedite/blob/master/scripts/lib/xpedite/profiler/__init__.py#L192) commands and they also return the loaded profileInfo instance as part of return value. To be consistent, I believe we should do the same here as well, else we will end up loading it twice, once in line 106 (new patch) and one more time inside `Profiler.probes(...)`.

Would you mind altering `[Profiler.probes()](https://github.com/Morgan-Stanley/Xpedite/blob/master/scripts/lib/xpedite/profiler/__init__.py#L209)` to return loaded profileInfo as return value. We can then change line 107 to
`profileInfo, probes = Profiler.probes(profileInfoPath)`.

Thanks again for the patch.

Reply:
Done the above changes.